SONAR-netCDF4/docs/crr341.adoc

```diff
@@ -415,7 +415,85 @@ Type 4 conversion equations are intended for data produced by Simrad EK80 and ES
+
+There are five right-handed Cartesian coordinate systems (<<coordinateSystemList>>) and one Cartesian angular coordinate system used to specify the physical location of received echoes. Some of the coordinate systems have different origins and this is implemented as a translation vector in the input coordinate system. Rotation angles are as per the right-hand rule.
+
+
+.List of coordinate systems used to physically locate the position of received echoes.
+[[coordinateSystemList]]
+
+|===
+|Coordinate system name|Origin|_x_-axis|_y_-axis|_z_-axis
+
+|Fixed coordinate system|Fixed somewhere on the nominal sea surface|Pointing North|Pointing East|Pointing down with gravity
+|Surface coordinate system|Platform origin|Forward along horizontal projection of platform main axis|Pointing horizontally to starboard orthogonal to horizontal projection of platform main axis|Pointing down with gravity
+|Platform coordinate system|Platform origin|Parallel to the main axis of the platform, positive values toward the front of the platform|Perpendicular to the main axis of the platform, positive values to the starboard side|Pointing down parallel to platform mast
+|Transducer coordinate system|Centre of transducer face|Pointing forward along transducer plane|Pointing starboard along transducer plane|Pointing down orthogonal to transducer plane
+|Sonar beam coordinate system|Centre of transducer face|Arbitrary|Orthogonal to the _x_-axis|Pointing along the beam axis
+|===
+
+The surface coordinate system (SCS) is obtained from the fixed coordinate system (FCS) through a translation of the origin (e.g., latitude and longitude assuming the FCS has latitude of 0 degrees and longitude of 0 degrees) and a rotation about the _z_-axis (typically the platform heading).
+
+The platform coordinate system (PCS) is obtained from the SCS through a rotation that corresponds to the pitch, roll of the platform. No translation is necessary.
+
+The transducer coordinate system (TCS) is obtained from the PCS through a translation of the PCS origin and a rotation to align with the transducer face.
+
+The sonar beam coordinate system (BCS) is obtained from the TCS through a rotation that specifies any difference in transducer pointing direction and beam pointing direction. In many cases there will be no rotation necessary. Note that split-aperture coordinates (minor and major angles) are defined in terms of the BCS and a rotation may be necessary.
+
+If a sonar actively alters beam pointing directions to compensate for motion of the platform, the rotations to get from the TCS to the BCS will be identical to those given to get from SCS to PCS, but with opposite sign (_is this correct? should there be a flag to avoid having to store these pitch, roll, heading values twice?_).
```

Comment:
I thinks it is clearer to have two sets of transformation matrix, and use the variable indicating if beam angle are compensated or not for motion (ie are they relative to the vertical or are they relative to the transducer face).